metron-analytics/metron-profiler-spark/README.md

```diff
@@ -263,6 +265,18 @@ The path to the input data read by the Batch Profiler.
 
 The format of the input data read by the Batch Profiler.
 
+### `profiler.batch.input.begin`
+
+*Default*: undefined
+
+Only messages with a timestamp after this will be profiled. By default, no time constraint is defined. The value is expected to follow the [ISO-8601 instant format](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT); 2011-12-03T10:15:30Z.
+
+### `profiler.batch.input.end`
```

Comment:
Just to be extra clear, is this a < or a <=?

Reply:
I updated the README to clarify this point.